pkg/app/handlers/users/handler.go

```diff
@@ -251,3 +234,35 @@ func (u UsersResourceHandler) setUserMappings(ctx context.Context, userID string
 	}
 	return nil
 }
+
+func (u UsersResourceHandler) getIdentityRelation(userID, identity string) *dsc.Relation {
+	rel := strings.Split(u.cfg.SCIM.IdentityRelation, "#")
+	if len(rel) == 2 {
```

Comment:
This logic is missing validation of the provided config.
* What if there are more than 2 '#' characters. Maybe use [strings.Cut](https://pkg.go.dev/strings#Cut) instead of Split.
* What if there are none?
* What if "IdentityRelation" is empty?